packages/playground/cli/tests/blueprints-v2.spec.ts

```diff
@@ -1,47 +0,0 @@
-import { loadNodeRuntime } from '@php-wasm/node';
-import { type PHPRequestHandler } from '@php-wasm/universal';
-import { bootRequestHandler } from '@wp-playground/wordpress';
-import { rootCertificates } from 'node:tls';
-import { runBlueprintV2 } from '@wp-playground/blueprints';
-import { RecommendedPHPVersion } from '@wp-playground/common';
-
-describe('V2 runner', () => {
-	let handler: PHPRequestHandler;
-
-	beforeEach(async () => {
```

Comment:
Do we have any other test confirming Blueprints v2 continue to work with the CLI runner? If not, we need to have something in place before removing this test.

Reply:
Ah, then we're good.